competitions/template/ansible/inv/local/group_vars/all.tmpl

```diff
@@ -0,0 +1,71 @@
+---
+competition_name: <COMPETITIONNAME>
+
+# The sheet that this wiki is going to act on.  Most of the time
+# this will be the same as the competition name (above), but sometimes
+# this is different when either the competition name doesn't work as
+# a sheet name (for instance, if it starts with a number), or if
+# this wiki should act on a different sheet.
+torquedataconnect_sheet_name: <COMPETITIONNAME>
```

Comment:
Suggest changing the value here to "`<USUALLY_COMPETITION_NAME>`" for clarity (and to make it more clearly match the same var in other config files -- I'll comment about that again later on).

Reply:
And in any case, an underscore between "COMPETITION" and "NAME" would be good.

Reply:
I'll make this change everywhere.